metar/Metar.py

```diff
@@ -596,15 +596,17 @@ def _handleRunway( self, d ):
           . name  [string]
           . low   [distance]
           . high  [distance]
-      """
-      if d['name']:
-          name = d['name']
-          low = distance(d['low'])
-          if d['high']:
-              high = distance(d['high'])
-          else:
-              high = low
-          self.runway.append((name,low,high))
+          . unit  [string]
+      """
+      if d['name'] is None:
+          return
+      unit = d['unit'] if d['unit'] is not None else 'FT'
```

Comment:
might be nice to condense this to 

```python
unit = d['unit'] or 'FT'
```

But that would change the logic if `d['unit']` was any other falsey value like `0`, `''`, or `[]`

Reply:
thanks, I'll stick with what I did for now.  For version 2.0, I think we should overhaul all the unit handling anyway.